FreeRTOS_Sockets.c

```diff
@@ -2551,7 +2551,11 @@ uint32_t FreeRTOS_inet_addr( const char * pcIPAddress )
     uint32_t ulReturn = 0UL;
 
     /* inet_pton AF_INET target is a 4-byte 'struct in_addr'. */
-    ( void ) FreeRTOS_inet_pton4( pcIPAddress, &( ulReturn ) );
+    if( pdFAIL == FreeRTOS_inet_pton4( pcIPAddress, &( ulReturn ) ) )
+    {
+        /* Return 0 if translation failed. */
+        ulReturn = 0;
```

Comment:
It is good to test the result of `FreeRTOS_inet_pton4()`, although it must be said the `ulReturn ` will be cleared in case of an error.

PS. MISRA likes an unsigned const : `ulReturn = 0U;`

Reply:
Thank you for the review @htibosch. I have update the value as you mentioned above.